demo/api/src/common/blocks/text-link.block.ts

```diff
@@ -2,4 +2,4 @@ import { createTextLinkBlock } from "@comet/blocks-api";
 
 import { LinkBlock } from "./linkBlock/link.block";
 
-export const TextLinkBlock = createTextLinkBlock({ link: LinkBlock });
+export const TextLinkBlock = createTextLinkBlock({ link: LinkBlock, name: "DemoTextLink" });
```

Comment:
This shouldn't work, name is the second argument of the function, and not part of the first one.